pyuvdata/uvdata.py

```diff
@@ -3150,3 +3150,122 @@ def _calc_single_integration_time(self):
             is in units of days, and integration_time has units of seconds, so we need to convert.
         """
         return np.diff(np.sort(list(set(self.time_array))))[0] * 86400
+
+    def get_antenna_redundancies(self, tol=1.0, include_autos=True):
+        """
+        Convenience method for getting baseline redundancies to a given tolerance
+        from antenna positions.
+
+        Note that the baseline numbers returned will only correspond with those in the file under the
+        u-positive convention.
```

Comment:
Maybe mention here that there's a function to reorder to get all of them? Or add an option that will call that function before calling the `get_antenna_redundancies` function.